### DISCUSSION TITLE
Vault belonging to an organisation under All Vaults empty

### DISCUSSION THREAD

**Discussion by mw-cyrano** [dani-garcia/vaultwarden#4424] [Q&A]
### Vault belonging to an organisation under All Vaults empty

We set up an organisation, which also has a vault. The folder structure and entries are accessible in the tab “organisations“, but not in the tab vaults. It says no entry.
![grafik](https://github.com/dani-garcia/vaultwarden/assets/90914631/853bbeac-3da9-4d6d-b726-8c1759456de0)

### Deployment environment

* Install method:

* Clients used: Web

### Steps to reproduce
Create an organisation with a vault.

### Expected behaviour
Entries are shown on the main tab when selecting the vault.

### Actual behaviour
No entries, see screenshot attached

**Comment by BlackDex** [comment]
Make sure you as an admin/owner have enabled access to the collections.
If  you do not explicitly give an admin/owner access to the collection(s) they will not be shown on the personal vault overview but only via the organization overview. This is by design.
